src/xrAICore/Navigation/graph_engine.h

```diff
@@ -109,4 +109,6 @@ class CGraphEngine
 #endif
 };
 
+#include "xrAICore/Navigation/PathManagers/path_manager.h"
+#include "xrAICore/Components/problem_solver_inline.h"
```

Comment:
It's not a good idea to remove 'inline' headers from 'base' headers with including 'inline's in different files. The trick with 'inline' headers is used in many places of the engine and your changes may mislead other people. Please try to find a better solution. 
P.S. The same changes were suggested in PR #299.

Reply:
And I'm don't remove 'inline' headers from 'base' headers with last commit:
https://github.com/OpenXRay/xray-16/pull/299/commits/70e5cd5274d8282bc863968c65873dc5f98540c9